osbs/build/build_request.py

```diff
@@ -1506,12 +1507,17 @@ def render(self, api=None, validate=True):
                 self.spec.build_image.value
 
         repo_name = git_repo_humanish_part_from_uri(self.spec.git_uri.value)
+        repo_dirname = git_repo_dirname_from_uri(self.spec.git_uri.value)
+        # Attempt to handle multiple instances of repos with the same name
+        # by tracking the directory they are in.
         # NOTE: Since only the repo name is used, a forked repos will have
         # the same git-repo-name tag. This is a known limitation. If this
         # use case must be handled properly, the git URI must be taken into
         # account.
         self.set_label('git-repo-name', repo_name)
         self.set_label('git-branch', self.spec.git_branch.value)
+        if repo_dirname:
+            self.set_label('git-dir-name', repo_dirname)
```

Comment:
To avoid ambiguity, can we fallback `git-dir-name` to an empty string if a `repo_dirname` is not set?